rasa_nlu/training_data/training_data.py

```diff
@@ -5,65 +5,29 @@
 from __future__ import print_function
```

Comment:
I don't think it is a good idea to have a file `training_data` in a module `training_data`